components/app/ColorMode.vue

```diff
@@ -0,0 +1,43 @@
+<script lang="ts" setup>
+const colorMode = useColorMode()
+
+const themes = [
+  // {
```

Comment:
delete?

Reply:
maybe not? so in the future we can add coffee mode and etc. ?